e2e/src/pages/app.po.ts

```diff
@@ -54,12 +56,26 @@ export class AppPage {
   async waitForElement(selector: Locator) {
     await browser.wait(async () => {
       return (await element(selector)).isPresent();
-    }, 10000, 'failed to load search results');
+    }, 30000, 'failed to load search results');
   }
 
   async waitForElementsArray(selector: Locator) {
     await browser.wait(async () => {
       return (await element.all(selector).isPresent());
-    }, 10000, 'failed to load search results');
+    }, 30000, 'failed to load search results');
+  }
+
+  async getItem(el: ElementFinder) {
+    const item = protractor.promise.defer();
+    browser.wait(function() {
```

Comment:
please use shorthand notation: `() => {`

Reply:
Agree with you...